Bottom: 94b90798440ff36bde247d4ea54ea446178dfa72
Top:    498065841b8d412ca17b37e89208362a7f81690d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:05 +0000

Adjust coordinate system GOODNES KNOWS WHETHER THE GEOMETRY IS CORRECT... IDA-STEP SEEMS TO COPE THOUGH


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a924326..bb63fa6 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -24,6 +24,7 @@
 #define PERFECT_ROUND_CONTOURS
 
 #define REVERSED_PCB_CONTOURS 1 /* PCB Contours are reversed from the expected CCW for outer ordering - once the Y-coordinate flip is taken into account */
+#undef REVERSED_PCB_CONTOURS
 
 #ifdef REVERSED_PCB_CONTOURS
 #define COORD_TO_STEP_X(pcb, x) (COORD_TO_MM(                   (x)))
diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 829221b..d47a8b4 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -26,6 +26,7 @@
 
 
 #define REVERSED_PCB_CONTOURS 1 /* PCB Contours are reversed from the expected CCW for outer ordering - once the Y-coordinate flip is taken into account */
+#undef REVERSED_PCB_CONTOURS
 
 #define EPSILON 1e-5 /* XXX: Unknown  what this needs to be */
